packages/cli/src/utils.ts

```diff
@@ -35,11 +37,14 @@ function getConfigDirectory(config: Config) {
   return config.configFile ? dirname(config.configFile) : process.cwd();
 }
 
-function isNotEmptyArray(args?: string[]): boolean {
+function isNotEmptyArray(args?: any[]): boolean {
```

Comment:
Let it work with any type.

```suggestion
function isNotEmptyArray<T>(args?: T[]): boolean {
```